ocaml/doc/OMakefile

```diff
@@ -23,7 +23,20 @@ sdk-install: doc-html
 	cp *.js *.html *.css *.png $(SDK)/docs/html/xenserver/
 	cp xenserver/* $(SDK)/docs/html/xenserver/
 
-noarch-install: sdk-install
+.PHONY: doc-html-install
+doc-html-install: doc-html
+	mkdir -p $(DESTDIR)$(DOCDIR)/html/xcp
+	sh -c 'cp -r -L api $(DESTDIR)$(DOCDIR)/html/xcp/'
```

Comment:
Why do we need `sh` here? Each line in a Makefile is executed by an independent shell. I assume it is the same for `omake`. Why is it not enough to have `cp -r -L api $(DESTDIR)$(DOCDIR)/html/xcp/`? There are more instances of this below.